cfep-20.md

```diff
@@ -0,0 +1,97 @@
+
+<table>
+<tr><td> Title </td><td> Standard package split (dev / doc / dbg) </td>
+<tr><td> Status </td><td> Proposed </td></tr>
+<tr><td> Author(s) </td><td> Sylvain Corlay &lt;sylvain.corlay@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> Oct 21, 2020</td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+This CFEP proposes a policy on how to split packages in multiple outputs for
+
+ - the main runtime requirements (shared objects, executables)
+ - the developer requirements (headers, static libraries, pkg-config and cmake artifacts)
+ - the requirements for debugging (debug symbols, JS map files)
+ - documentation.
+
+Notes:
+
+ - The Debian community follows a convention similar to this proposal.
+ - Should this CFEP be accepted, it would supersede [CFEP 18](https://github.com/conda-forge/cfep/blob/master/cfep-18.md) on the split of static libraries.
+ - This CFEP will be much easier to execute upon if we improve the conda tooling around the support for multiple outputs for recipes.
+
+## Motivation
+
+At the moment, there is not a common pattern to split packages in multiple output including C/C++ headers, JS source maps, or artifacts of build systems that are not required at runtime but are necessary at build time. Packages have been split in various ways, but adopting a convention will allow us to create tooling, idioms, and best practices on how to deal with such split packages.
```

Comment:
Would it be possible to provide a more detailed motivation section?
What are the benefits that we would reap from the proliferation of split packages?
Could we obtain these benefits without split packages?

Reply:
> Would it be possible to provide a more detailed motivation section?

Yes @CJ-Wright, I will expand a bit on the motivation section today.